onshape/onshape/postprocess.py

```diff
@@ -136,6 +136,9 @@ async def postprocess(
     paths = [urdf_path]
     if config.add_mjcf:
         for metadata in config.mjcf_metadata or []:
+            # Would like to avoid this, but how?
+            metadata.joint_name_to_metadata = config.joint_metadata
+            metadata.actuator_type_to_metadata = config.actuators
```

Comment:
i would just add `JointMetadata` and `Actuators` as args to `convert_urdf_to_mjcf` instead of overloading the config like this...